stdlib/universe/mode.go

```diff
@@ -240,44 +240,37 @@ func (t *modeTransformation) doString(cr flux.ColReader, tbl flux.Table, builder
 		}
 	}
 
-	if numNil > max {
+	if numNil == total {
```

Comment:
I guess this confuses me. If we're not considering null values, why do we need to count the null values? Is the intent here that if there are no values that it will emit null?

I'd like a comment on this and I think this could be potentially better done by appending nil if there were no values considered.